src/engraving/playback/playbackmodel.h

```diff
@@ -109,7 +109,7 @@ class PlaybackModel : public async::Asyncable
                       ChangedTrackIdSet* trackChanges = nullptr);
 
     void processSegment(const int tickPositionOffset, const Segment* segment, const std::set<staff_idx_t>& staffIdxSet,
-                        ChangedTrackIdSet* trackChanges);
+                        ChangedTrackIdSet* trackChanges, bool isFirstSegmentOfMeasure);
```

Comment:
Since trackChanges here is an output argument, I would suggest changing the order of the arguments:
processSegment(..., isFirstSegmentOfMeasure, trackChanges)